src/assets/locales/de/main.json

```diff
@@ -1066,6 +1066,18 @@
       "metadata_url_required_error": "Metadaten-URL ist erforderlich",
       "metadata_url_invalid_error": "Bitte geben Sie eine gültige Metadaten-URL ein"
     },
+    "clearinghouseSelfDescription": {
+      "heading": "Status des Selbstbeschreibungsdokuments",
+      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Selbstbeschreibungen für Administratoren.",
+      "reprocess": "Wiederaufbereiten",
+      "complianceStatus": "Compliance-Status:",
+      "legalPerson": "Juristische Person",
+      "legalPersonDesc": "Übersicht über Unternehmen mit fehlender Selbstbeschreibung einer juristischen Person",
+      "connectors": "Anschlüsse",
```

Comment:
as discussed with @MaximilianHauer , please change to "Connectors", in German as well

Reply:
Changes Done